php/Dockerfile

```diff
@@ -1,68 +1,53 @@
-FROM php:7.1-fpm
-
-MAINTAINER Alexandre JARDIN <aja@emakina.fr>
-LABEL maintainer="Alexandre JARDIN <aja@emakina.fr>"
+FROM php:7.1-fpm-alpine
 
 # Install Magento requirements
 RUN \
-    apt-get update -qq && \
-    apt-get install -y --no-install-recommends \
-        bzip2 \
+    apk add --no-cache --virtual .persistent-deps \
+        freetype-dev \
         git \
-        gnupg \
-        libcurl4-openssl-dev \
-        libicu-dev \
-        libfreetype6-dev \
-        libjpeg-dev \
+        icu-libs \
+        libjpeg-turbo-dev \
         libmcrypt-dev \
         libpng-dev \
         libxml2-dev \
+        libxml2-utils \
         libxslt-dev \
-        ssmtp && \
+        openssh-client \
+        perl \
```

Comment:
Wouldn't it be useful to add the git patch dependency here ?

Reply:
Good catch!